build.cake

```diff
@@ -482,17 +478,17 @@ Task("BuildChocolateyPackages")
                 }
             });
 
-        ChocolateyPack("choco/nunit-console-with-extensions.nuspec",
-            new ChocolateyPackSettings()
-            {
-                Version = productVersion,
-                OutputDirectory = PACKAGE_DIR,
-                Files = new [] {
-                    new ChocolateyNuSpecContent { Source = CURRENT_IMG_DIR + "LICENSE.txt", Target = "tools" },
-                    new ChocolateyNuSpecContent { Source = CURRENT_IMG_DIR + "NOTICES.txt", Target = "tools" },
-                    new ChocolateyNuSpecContent { Source = CHOCO_DIR + "VERIFICATION.txt", Target = "tools" }
-                }
-            });
+        //ChocolateyPack("choco/nunit-console-with-extensions.nuspec",
+        //    new ChocolateyPackSettings()
+        //    {
+        //        Version = productVersion,
+        //        OutputDirectory = PACKAGE_DIR,
+        //        Files = new [] {
+        //            new ChocolateyNuSpecContent { Source = CURRENT_IMG_DIR + "LICENSE.txt", Target = "tools" },
+        //            new ChocolateyNuSpecContent { Source = CURRENT_IMG_DIR + "NOTICES.txt", Target = "tools" },
+        //            new ChocolateyNuSpecContent { Source = CHOCO_DIR + "VERIFICATION.txt", Target = "tools" }
+        //        }
+        //    });
```

Comment:
Should this be removed?

Reply:
Definitely... I'm afraid I'm jumping back and forth across too many projects and missing stuff!!!